Tests/PostgresNIOTests/New/Extensions/PSQLBackendMessage+Equatable.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 02.02.21.
+//
+
+@testable import PostgresNIO
```

Comment:
Ditch Xcode template headers.
```suggestion
@testable import PostgresNIO
```